loops.js

```diff
@@ -33,6 +41,13 @@
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over. Google daarom eens naar 'mdn webdocs remainder operator'.
 // ==========================================
 
+for (let i = 0; i < 16; i++) {
+    if (i % 2){
```

Comment:
In je if statement mis je dus je operator, wanneer dus de i deelt door twee en er niets over blijft....i % 2 === 0.
Hier dien je tevens twee if statements te hebnben ipv if else...